frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -764,9 +773,38 @@ export class TileViewPage extends React.Component {
     );
   }
 
+  renderItems(items, renderTile) {
+    return (
+      <Gallery gutter="sm" className="catalog-tile-view-pf catalog-tile-view-pf-no-categories">
+        {_.map(items, (item) => (
+          <GalleryItem key={item.uid ? `gallery-${item.uid}` : `gallery-${item.obj.metadata.uid}`}>
+            {renderTile(item)}
+          </GalleryItem>
+        ))}
+      </Gallery>
+    );
+  }
+
+  renderGroupedItems(items, groupBy, renderTile, groupItems) {
+    const groupedItems = groupItems(items, groupBy);
+    return _.map(
+      groupedItems,
+      (value, key) =>
+        value.length > 0 && (
+          <>
+            <Title className="co-catalog-page__group-title" headingLevel="h2" size="lg">
+              {key} ({_.size(value)})
+            </Title>
+            {this.renderItems(value, renderTile)}
+            <br />
+          </>
```

Comment:
I think this can be better represented with css rather than a `br`. I think `br`s are old fashioned and we lose the fidelity to control the gap. 

Also, you lack the `key` prop. Anytime you do `map` you need to use the `key` prop on returned JSX.

Reply:
- added key prop 
- removed <br/> and added a css class to handle the space between the groups